<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1760</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система Учета Кадров</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditSearch">
        <property name="placeholderText">
         <string>Поиск...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSearch">
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFilter">
        <property name="text">
         <string>Фильтрация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="text">
         <string>Добавить Сотрудника</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>Удалить Сотрудника</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSettings">
        <property name="text">
         <string>Настройки</string>
        </property>
        <property name="icon">
         <iconset theme="preferences-system">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleRowSelection</enum>
      </property>
      <column>
       <property name="text">
        <string>Имя</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Фамилия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Отчество</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Паспорт</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Должность</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Мед.Полис</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Страховой Полис</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Отпуск</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="paginationLayout">
      <item>
       <widget class="QLabel" name="labelRowCount">
        <property name="text">
         <string>Строк: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCurrentPage">
        <property name="text">
         <string>Страница: 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonPrevPage">
        <property name="text">
         <string>Предыдущая страница</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNextPage">
        <property name="text">
         <string>Следующая страница</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>